lib/Doctrine/Common/Collections/Expr/ClosureExpressionVisitor.php

```diff
@@ -46,11 +46,13 @@ public static function getObjectFieldValue($object, $field)
             return $object[$field];
         }
 
-        $accessors = array('get', 'is');
+        $accessors = array(
+            'get' . $field,
+            'is' . $field,
+            $field, // non-standard method
```

Comment:
The only change required here would've been just adding empty string to the end of the list. The rest can stay the same. The effect would've been the same.